rules/DowngradePhp80/Rector/Expression/DowngradeMatchToSwitchRector.php

```diff
@@ -156,7 +154,7 @@ private function createSwitchStmts(Echo_ | Expression | Return_ $node, MatchArm
         } elseif ($node instanceof Echo_) {
             $stmts[] = new Echo_([$matchArm->body]);
             $stmts[] = new Break_();
-        } else if ($node->expr instanceof MethodCall || $node->expr instanceof FuncCall) {
+        } elseif ($node->expr instanceof CallLike) {
```

Comment:
Here should be probably any `Expr`, e.g. `echo` etc.

Reply:
`Echo_` is a statement, already handled before, and this check is fill against its `args` property of object, which only exists on class child of `CallLike`, I moved the check in the last elseif

Reply:
I see :+1: